ibm_catalog.json

```diff
@@ -58,9 +57,7 @@
             "diagrams": [
               {
                 "diagram": {
-                  "url_proxy": {
-                    "url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/blob/main/assets/architecture.svg"
-                  },
+                  "url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/blob/main/assets/architecture.svg",
```

Comment:
I believe this needs to be the raw url. EG: `https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da/main/assets/architecture.svg`

Reply:
the old url_proxy worked also without raw url, I'll fix this. Thanks!

Reply:
thats expected when you open it directly in a browser. It will show in catalog UI

Reply:
done